framework/src/main/java/org/checkerframework/framework/qual/LiteralKind.java

```diff
@@ -11,8 +11,7 @@
  * that enum is in the tools.jar which is not on the user's classpath by default. So this enum is
  * used instead.
  *
- * @checker_framework.manual #creating-type-introduction Declaratively specifying implicit
- *     annotations
+ * @checker_framework.manual #creating-type-introduction Declaratively specifying annotations
```

Comment:
Should this mention defaults?  The related section is about type introduction, not about specifying annotations per se.